biz.aQute.bnd.gradle/src/aQute/bnd/gradle/BndPlugin.groovy

```diff
@@ -49,6 +49,7 @@ public class BndPlugin implements Plugin<Project> {
           throw new GradleException("Unable to load bnd workspace ${rootDir}")
         }
       }
+      bndWorkspace.setOffline(project.gradle.startParameter.offline)
```

Comment:
This should be up inside the previous if block since we only need/want to set it IF we created the Workspace.